test/scripts/e2e_go_tests.sh

```diff
@@ -10,10 +10,12 @@ export GO111MODULE=on
 # Anchor our repo root reference location
 REPO_ROOT="$( cd "$(dirname "$0")" ; pwd -P )"/../..
 
-# Need bin-race binaries for e2e tests
-pushd ${REPO_ROOT}
-make build-race -j4
-popd
+if [ "${NORACEBUILD}" = "" ]; then
```

Comment:
instead of passing a global environment variable `NORACEBUILD`, use a script parameter to indicate that the script should not rebuild.

Reply:
Updated to use -norace argument.